test_util/cluster_api.py

```diff
@@ -274,6 +274,7 @@ def get_test_app(self, custom_port=False):
             'cpus': 0.1,
             'mem': 32,
             'instances': 1,
+            'user': 'nobody',
```

Comment:
Very explicitly do _not_ want to specify a user automatically and check what the default / no app user is for some of these cases (See https://mesosphere.atlassian.net/browse/DCOS-9040). That's why the user checking was added.